### DISCUSSION TITLE
This is too EP!

### DISCUSSION THREAD

**Discussion by ghost** [AmusementClub/vapoursynth-classic#1] [General]
![EDAEF92D-C831-438C-8BBA-8F7C36CE4C3E](https://user-images.githubusercontent.com/68379058/131789587-0728c891-6f04-4732-8377-bf21ecc0f1d5.jpeg)

**Comment by AkarinVS** [comment]
But I'm really tired of chasing daily incompatible changes and constantly
updating all my "weird" scripts.

>
